src/styles/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 300ms ease;
+  cursor: pointer;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    margin: 0 auto;
+    padding-top: 16px;
+
+    img {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__description {
+    margin-top: 36px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    transition: color 300ms ease;
+
+    &:hover {
+      color: $secondary-title-color;
+    }
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-text;
+  }
+
+  &__score {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+    color: $color-text;
+  }
+
+  &__price-label {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-top: 24px;
+
+    p {
+      margin: 0;
+      font-size: 12px;
+      font-weight: 400;
+      color: $secondary-text;
+    }
+
+    p:last-child {
```

Comment:
Don't use tag names for styling (except html and body)
![image](https://github.com/user-attachments/assets/1a3e6810-6430-4ca3-88c2-99e0b9f4ebf7)